dashboard/app/models/levels/blockly.rb

```diff
@@ -180,10 +180,20 @@ def self.count_xml_blocks(xml_string)
     PROCEDURE: 'Functions',
     VARIABLE: 'Variables',
   }
+
+  GOOGLE_BLOCKLY_NAMESPACE_XML = "<xml xmlns=\"https://developers.google.com/blockly/xml\">"
+
+  # This function converts category blocks used by levelbuilders to edit the toolbox to category tags
+  # and places the appropriate blocks within each category
   def self.convert_toolbox_to_category(xml_string)
+    is_google_blockly = false
+    if xml_string.include? GOOGLE_BLOCKLY_NAMESPACE_XML
+      is_google_blockly = true
+      xml_string[GOOGLE_BLOCKLY_NAMESPACE_XML] = "<xml>"
```

Comment:
This seems reasonable and probably easier than updating the xpath logic to expect both variations of the xml tag.